CHANGELOG.md

```diff
@@ -1,5 +1,41 @@
 # Changelog
 
+## 0.12.6 <Badge text="beta" type="success" />
+
+Released on July 28, 2020.
+
+### Features
+
+- Add `flat` operator for unnesting and flat-maps - [#2898](https://github.com/PrefectHQ/prefect/pull/2898)
```

Comment:
```suggestion
- Add `flatten` operator for unnesting and flat-maps - [#2898](https://github.com/PrefectHQ/prefect/pull/2898)
```

Just realized this oversight when you posted the release changelog (which I updated)!